OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -297,7 +297,11 @@ yield return ((IModifyableRenderable)r)
 							.WithAlpha(shadowAlpha);
 				}
 
-				var palette = wr.Palette(info.Palette + (info.IsPlayerPalette ? args.SourceActor.Owner.InternalName : ""));
+				var paletteName = info.Palette;
+				if (paletteName != null && info.IsPlayerPalette)
+					paletteName += args.SourceActor.Owner.InternalName;
+
+				var palette = wr.Palette(paletteName);
```

Comment:
Do we even have to call this when `paletteName` is `null`? (I doubt it. But on the otherhand I don't know if it is worth the extra code for the null case...)

Reply:
Well the explaining null handling comment is in wr.Palette() so i kept it here as-is to avoid copying the explaining comment.